modules/Kconfig.simplelink

```diff
@@ -29,3 +29,7 @@ config HAS_MSP432P4XXSDK
 
 config HAS_CC13X2_CC26X2_SDK
 	bool
+
+config HAS_CC13X2X7_CC26X2X7_SDK
+	bool
+
```

Comment:
@jadonk can you please drop this newline at EOF.

Reply:
k.